src/components/LoadingScreen/LoadingScreen.js

```diff
@@ -1,22 +1,76 @@
 // @flow
-import React, { PureComponent } from 'react';
+import React, { Fragment, PureComponent } from 'react';
```

Comment:
nit: Fragment is unused

Reply:
Oh, OK, I missed that one. I'll remove it.